build.cake

```diff
@@ -554,12 +554,24 @@ Task("SetPackageVersions")
 Task("SRGen")
 	.Does(() =>
 {
+    // save current working directory to restore at end of task
+    var taskStartedWorkingDirectory = System.IO.Directory.GetCurrentDirectory();
+    var relativePathPrefix = "." + System.IO.Path.DirectorySeparatorChar.ToString();
```

Comment:
You shouldn't need to add the separator char here - just using "." in Path.Combine should work fine.

Reply:
Good point - will fix. FYI I'm porting this over from @kenvanhyning 's fix in other repo, so we should likely update there too.